### ISSUE TITLE
Inconsistent formatting of command flags with arguments

### ISSUE THREAD

**Issue by richsalz** [openssl/openssl#9987]
We're not consistent in documenting command line flags that take arguments:

1 ```[B<-salt> I<string>]```
1 ```B<-inform SMIME|PEM|DER>]```
1 ```[B<-I<cipher>>]``` 

I think the last two cases should all be formatted like the first.  Thoughts?

**Comment by levitte** [comment]
The first choice is consistent with what's being said about formatting in [man-pages(7)](http://man7.org/linux/man-pages/man7/man-pages.7.html), so yes.

**Comment by richsalz** [comment]
I will fix this after my other doc-related PR's get merged: #9953, #9986, #9974.

**Comment by richsalz** [comment]
Does this also hold for ```=item``` entries?  Like
```    =item B<-section foo>```
should be
 ```  =item B<-section> I<foo>```

?

**Comment by levitte** [comment]
Yes. You should also be careful with | when used to indicate alternatives. They should not have any special formatting, so items like `DER|PEM` should be marked up `B<DER>|B<PEM>`

**Comment by richsalz** [comment]
I fixed all of the issues raised so far, and changed all ```B<...|...>``` or ```I<nm:v>``` as mentioned, even if you didn't specifically mention.

I tried to think of a perl regexp that would catch the misformed ones, but couldn't.

**Comment by richsalz** [comment]
I mean, "coudn't without having too many false positives"

**Comment by levitte** [comment]
> I mean, "coudn't without having too many false positives"

Yup...  you can only do so much before you have to implement a full fledged AI.  I'm sure you're looking forward to the moment.  (I totally expect a perl implementation :wink:)

**Comment by richsalz** [comment]
I think I can avoid Skynet by using a non-greedy match for ```B<([^>]+)>``` and then seeing if ```$1``` has bad characters.  Gonna try it out now.  If it goes wrong, please send someone from the future to comment on this PR.

**Comment by richsalz** [comment]
This gives me errors:
```
    while ( $contents =~ /([BI])(<[^>]*>)/g ) {
        err($id, "Bad content inside $1<$2>")
            if $2 =~ /[<|]/;
    }       

Use of uninitialized value $1 in concatenation (.) or string at ./util/find-doc-nits line 267.
Use of uninitialized value $2 in concatenation (.) or string at ./util/find-doc-nits line 267.
```

**Comment by richsalz** [comment]
oh of course I have to save the variables before the err/if call.

**Comment by richsalz** [comment]
Fixed, updated find-doc-nits, fixed the problems it found.  The changes in ```BIO_ADDR.pod``` might be controversial, please take a look.  The other option is to remove the embedded ```L<>``` tag, or allow them, or something else.

**Comment by levitte** [comment]
What `BIO_ADDR.pod` changes are you talking about?

**Comment by richsalz** [comment]
oh, oops, sorry it's in #10022

**Comment by richsalz** [comment]
I removed the "Fixes: 9987" from the commits that are in #10022, since it is clear in the comments that there is still more to do.

**Comment by richsalz** [comment]
I think the combination of #10022 and various PR's that @levitte did mean that this is fixed now.
